app/controllers/lists_controller.rb

```diff
@@ -0,0 +1,45 @@
+class ListsController < ApplicationController
+
+  # users must be signed in before any lists_controller method
+  before_action :authenticate_user!
+
+  def index
+    @lists = List.all
+  end
+
+  def show
+    @list = List.find(params[:id])
```

Comment:
Understood, but every time I add that line in, I get a noMethodError in the show.html.erb on line 1, stating that the method `title` is undefined, which is confusing me.